keyboards/latinpadble/rules.mk

```diff
@@ -1,29 +1,17 @@
-
 # MCU name
 MCU = atmega32u4
 
 # Bootloader selection
-#   Teensy       halfkay
-#   Pro Micro    caterina
-#   Atmel DFU    atmel-dfu
-#   LUFA DFU     lufa-dfu
-#   QMK DFU      qmk-dfu
-#   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
 BOOTLOADER = caterina
 F_CPU = 8000000
 
-
 # Feather uses the caterina bootloader
 OPT_DEFS += -DBOOTLOADER_SIZE=4096 -DCATERINA_BOOTLOADER
```

Comment:
```suggestion
```

Reply:
I felt like this was unnecessary, but I couldn't find where it was set in core.

Reply:
`bootloader.mk`

Reply:
Ahh, it was in the root directory; that's why I didn't find it.